ddpui/ddpairbyte/airbyte_service.py

```diff
@@ -115,50 +118,50 @@ def abreq(endpoint, req=None, **kwargs):
     return {}
 
 
-def get_workspaces():
+def get_workspaces() -> Dict:
```

Comment:
@AyushDineshRathi we want specific dataclasses defining each key in the response. This is a generic type annotation that doesn't help a lot. 

So you will need to refer to the airbyte api spec and see what it is returning.